### ISSUE TITLE
Sometimes, instance deleting does not work: Internal Server Error

### ISSUE THREAD

**Issue by parmentf** [Inist-CNRS/ezmaster#20]
![image](https://user-images.githubusercontent.com/595509/27482708-ca1c34b8-5822-11e7-8f86-6358cf4d9705.png)

**Comment by kerphi** [comment]
without server side logs (``docker logs ezmaster``) or a way to reproduce it, it will be hard do fix.

**Comment by parmentf** [comment]
Yep. It's systematically on the same machine. We will reproduce.
